package.nls.de.json

```diff
@@ -0,0 +1,27 @@
+{
+  "command.regenerateIcons.title": "Icon-Veränderungen anwenden",
+  "command.ngPreset.title": "Angular Preset umschalten (Workspace Level)",
+  "command.jsPreset.title": "Offizielles JS Preset umschalten (User Level)",
+  "command.tsPreset.title": "Offizielles TS Preset umschalten (User Level)",
+  "command.jsonPreset.title": "Offizielles JSON Preset umschalten (User Level)",
+  "command.hideFoldersPreset.title": "Ordner-Icons Sichtbarkeit umschalten (User Level)",
+  "command.foldersAllDefaultIconPreset.title": "Spezifische Order-Icons umschalten (User Level)",
+  "command.restoreIcons.title": "Icon-Manifest zurücksetzen",
+  "command.resetProjectDetectionDefaults.title": "Projekterkennung zurücksetzen",
+  "configuration.title": "vscode-icons Einstellungen",
+  "configuration.dontShowNewVersionMessage.description": "Wenn aktiviert, zeigt die Erweiterung keine Benachrichtigung nach dem Installieren einer neuen Version an",
```

Comment:
`Wenn aktiviert` means `When enabled`.
This raises the following question. `How do I enable it?`
We must be specific of what the value has to be. 
Consider that we are talking to a 10-year-old or to someone that is just starting with development.

Proposal: `Wenn auf 'true' gesetzt, ...`